CONTRIBUTING.md

```diff
@@ -39,35 +39,94 @@ Check the "Developer Setup" below for the developer workflow.
 
 ## 2. 🔨 Developer Setup
 
-First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and add to path by going to advanced system settings. Then, launch cmd.
+First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and follow the recommended settings. Once installed, open up the "Anaconda Prompt" and type in the command "conda --version". If everything has worked, it should print "conda xx.xx.x".
+For example: "conda 23.11.0"
+
+(P.S: You may add Anaconda to your path environment variables on your own [if you feel comfortable], however this is discouraged by Anaconda themselves as it may "interfere with other software". Instead, they recommend to use the Anaconda Prompt.)
```

Comment:
no nested parentheses or brackets

Reply:
where shd i specifically add the nested brackets? I added them in line 45 since I thought it was needed there "(P.S: You may add Anaconda to your path environment variables on your own [if you feel comfortable],"